Alameen688/MyDiary
# 159543280 Update develop branch with UI update

#### What does this PR do?
Delivers changes to ui template based on adjustment in design for frontend client
#### Description of Task to be completed?
The following should reflect
- color change from red to green for edit and add entry button
- Delete button removed from view-entry.html page
#### How should this be manually tested?
After cloning the repo, cd into it then
- go to UI/view-entry.html and notice the edit button color has changed
- and delete button has also been removed
#### Any background context you want to provide?
none
#### What are the relevant pivotal tracker stories?
[#159543280](https://www.pivotaltracker.com/story/show/159543280)
